src/server/pps/server/pipeline_controller.go

```diff
@@ -93,21 +99,21 @@ func (m *ppsMaster) newPipelineOp(ctx context.Context, cancel context.CancelFunc
 		etcdPrefix:   m.a.etcdPrefix,
 
 		bumpChan:  make(chan struct{}, 1),
-		opManager: m.om,
+		opManager: m.pcMgr,
 	}
-	return op
+	return pc
 }
 
-// Bump signals the pipelineOp goro to either process the  latest state
+// Bump signals the pipelineController goro to either process the  latest state
 // of the pipeline now, or once its done with its current processing iteration.
 //
 // This function is expected to be called while holding the opManager lock.
 // This helps guard the critical section when the goroutine is cleaned up
-// as part of a pipeline deletion event in pipelineOp.tryFinish().
+// as part of a pipeline deletion event in pipelineController.tryFinish().
 //
 // Note: since op.bumpChan is a buffered channel of length 1, Bump() will
 // add to the channel if it's empty, and do nothing otherwise.
-func (op *pipelineOp) Bump() {
+func (op *pipelineController) Bump() {
```

Comment:
would it make sense to change the receiver names from `op` to `pc` here as well?

Reply:
it would make so much sense! lol . I caught this after merging - I'll make another PR to change these